### DISCUSSION TITLE
Migrate settings from 4.2.0 to 4.3.3?

### DISCUSSION THREAD

**Discussion by daprice** [MonitorControl/MonitorControl#1697] [Q&A] [answered]
I had been using 4.2.0 and had to tweak a lot of settings to get my monitor to work well – I didn’t memorize all of the settings of course. When I installed 4.3.3, it seems to have set everything back to the defaults. I still see the old settings file in ~/Library/Preferences, but the new version doesn’t recognize the settings even if I rename the previous file to `app.monitorcontrol.MonitorControl.plist`.

Is there a way to get the new version to use my settings other than going back to the old version, taking screenshots of all the settings, and manually applying them to the new version?

**Comment by waydabber** [comment, marked as answer]
Yes, you should be able to use `defaults export` and `import` to export the old settings and import the new one. Renaming won't work as the settings are cached. But if you renamed the old file things might have gotten lost by now.
